docs/mqtt.md

```diff
@@ -594,6 +594,31 @@ to reconnect to switch to a new virtual host.
 is considered more specific than the port-to-vhost mapping with the `mqtt_port_to_vhost_mapping`
 global parameter and so takes precedence over it.
 
+#### Use client_id from client certificate to authenticate 
+
+By the default, the plugin passes the `client_id` it received from the MQTT connection to the configured authentication backends as a credential. However, it is possible to extract the `client_id` from the certificate in a very similar way as we extract the username. 
+
```

Comment:
Perhaps rephrase the 1st sentence to: By the default, the plugin passes the `client_id` as a credential to the configured authentication backends.

Do they need to know this < which it received from the MQTT connection> if not I would leave it out?

Reply:
For the 2nd sentence how about: "However, it is possible to extract the `client_id` from the certificate using the same way the username is extracted. "

Is is the same way or similiar? similar is a bit vague?

Reply:
You are right, we may leave that detail out. Thanks!

Reply:
i leave out "the same way " .. to avoid any confusion.